bin/agent-data-plane/src/config.rs

```diff
@@ -15,6 +15,10 @@ pub enum Action {
     /// Runs the data plane.
     #[command(name = "run")]
     Run(RunConfig),
+
+    /// Configures the log level of the data plane.
```

Comment:
Since this will end up eventually having more sub-subcommands related to debugging, better to say something generic, along the lines of `Various debugging commands.`, etc.